scripts/cypress.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if [ ${1:-run} = "fast" ]; then
```

Comment:
```suggestion
if [ ${1:-run} = "fast" ]; then
```
if [ "$1" = "fast"]; then

Reply:
(Noted below, but checked it here). Without the default it throws ` [: =: unary operator expected` on a command without an argument (`yarn cy`).

Reply:
That was true even with the quoted `"$1"`?

Reply:
Ahhhhh no I missed the quotes! Thank you! These changes look good with the quotes.